packages/block-library/package.json

```diff
@@ -27,7 +27,7 @@
 	"sideEffects": [
 		"build-style/**",
 		"src/**/*.scss",
-		"{src,build,build-module}/*/init.js"
+		"{src,build,build-module}/**/init.js"
```

Comment:
Just noting that this will expand the list of files that are allowed to have side effects so let's make sure that this change is intentional. I ran into the same thing in https://github.com/WordPress/gutenberg/pull/49305#discussion_r1151033211

Reply:
This has been changed. Thanks! :D